controller/custom_message_controller.py

```
@@ -0,0 +1,70 @@
+import os
+import json
+import logging
+import traceback
+from pathlib import Path
+
+from fastapi import status
+from fastapi.responses import JSONResponse
+from constants import MESSAGE_PATH
+
+logger = logging.getLogger("log")
+
+
+class CustomMessageClient:
+    @staticmethod
+    def get_custom_message():
+        cwd = os.getcwd()
+        relative_path = os.path.join(cwd, MESSAGE_PATH)
+        logger.info(f"Reading message from file: {relative_path}")
+        try:
+            with open(relative_path, "r") as file:
+                logger.info(f"Reading message from file: {relative_path}")
+                data = json.load(file)
+                logger.info(f"Message read from file: {data}")
+                message = data.get("message", "No message found")
+                return JSONResponse(
+                    status_code=status.HTTP_200_OK, content={"message": message}
+                )
+
+        except Exception as e:
```

For this alone incase of any issue while reading the file. send a custom message as response with 200.
I dont want to raise any exception here.

---

{"message": "Our Services"}